### PR TITLE
Running tests using python 3.12

### PR DESCRIPTION
**Pull request by salman2013**
**Description:**

Running tests for python 3.11 and 3.12

Ticket:  https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=53243529,
              https://github.com/openedx/axim-engineering/issues/1045

### CODE REVIEW

**Review by feanil — COMMENTED** [review]
Looks like everything is passing on the newer versions, can you update the classifiers in setup.py as a part of this PR as well? Then I think we can merge and release this.

**Comment by salman2013** [comment]
@feanil i have updated the classifiers.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ['3.8']
-        toxenv: [py38-django32, py38-django42, quality]
```
Comment (farhan) [inline .github/workflows/ci.yml:18]: Should we also remove the django32 support from all the repositories?

File: setup.py [hunk 2]
```diff
@@ -141,6 +141,9 @@ def get_version(file_path):
     classifiers=[
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 3',
```
Comment (farhan) [inline setup.py:143]: I think we should drop their support as well
All repositories should only give support for `v3.8`, `v3.11` and `v3.12`

File: .github/workflows/ci.yml (hunk 1 above)
Comment (salman2013) [reply .github/workflows/ci.yml:18]: yes, we are doing this along this task.

File: setup.py (hunk 2 above)
Comment (salman2013) [reply setup.py:143]: As per Feanil's direction we have to keep this.

Comment (farhan) [reply setup.py:143]: We should remove python 2 classifier as recommended here
https://openedx.slack.com/archives/C05P7KQURL0/p1708438058899899?thread_ts=1708432662.032769&cid=C05P7KQURL0

**Review by farhan — CHANGES_REQUESTED** [review]
1 change

File: setup.py (hunk 2 above)
Comment (salman2013) [reply setup.py]: @farhan I have fixed the above comment.

**Review by farhan — APPROVED** [review]

**Pull request merged**